templates/node/lib/client.js.twig

```diff
@@ -70,23 +71,39 @@ class Client {
         headers = Object.assign(this.headers, headers);
 
         let contentType = headers['content-type'].toLowerCase();
+
+        let formData = null;
+
+        // Compute FormData for axios and appwrite.
+        if (contentType.startsWith('multipart/form-data')) {
+            const form = new FormData();
+            
+            let test = this.flatten(params);
```

Comment:
Probably better use a more meaningful name here

Reply:
Oh, yeah. I forgot about that, that's when I was trying to figure it out by quickly prototyping new ideas

Reply:
I'll fix it now